packages/tizen_app_manager/tizen/src/tizen_app_manager_plugin.cc

```diff
@@ -7,63 +7,65 @@
 #include "tizen_app_manager.h"
 
 static bool AppInfoCB(app_info_h app_info, void *user_data) {
-  if (app_info != nullptr) {
-    TizenAppManagerPlugin *plugin = (TizenAppManagerPlugin *)user_data;
-    flutter::EncodableMap value;
-    int ret = application_utils::GetAppData(app_info, value);
-    if (ret == APP_MANAGER_ERROR_NONE) {
-      plugin->m_applications.push_back(flutter::EncodableValue(value));
-    }
+  if (app_info == nullptr) {
+    return false;
+  }
+  TizenAppManagerPlugin *plugin = (TizenAppManagerPlugin *)user_data;
```

Comment:
Please use C++ style type cast.

Reply:
it has been modified by using static_cast. Thank you!